solr/core/src/java/org/apache/solr/schema/IndexSchema.java

```diff
@@ -1991,6 +1997,37 @@ public IndexSchema deleteCopyFields(Map<String, Collection<String>> copyFields)
     throw new SolrException(ErrorCode.SERVER_ERROR, msg);
   }
 
+  /** Informs analyzers used by a fieldType. */
+  protected void informResourceLoaderAwareObjectsForFieldType(FieldType fieldType) {
```

Comment:
does this need to be protected? The inner method is private, so why is this one public? A method reference in lambdas does not need to be visible in any way.

Reply:
This is still also called from `ManagedIndexSchema` class which is a subclass.

Reply:
This is not ideal. If you move the call to this private method into `postReadInfom()` of *this* class (`IndexSchema`), then the subclass would also call it and it could be private.
Here it is called in readSchema *after* `postReadInform()`. If it is moved there both implementations automatically use it.